### PR TITLE
Update contact-pstn-service-desk.md

### PR DESCRIPTION
**Pull request by ErsancMS**
Updated information on contact info from the PSTN Service Desk to reflect updated contacts

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 1]
```diff
@@ -25,12 +25,16 @@ description: "When you are getting phone numbers or porting (transferring) numbe
...
 
 Then send your request to one of the following email addresses:
 
-- For help with phone numbers inside the U.S., [send your request](mailto:ptn@microsoft.com).
-- For help with phone numbers outside the U.S., [send your request](mailto:ptneu@microsoft.com).
+### USA Or Canada
+- For help with getting new phone numbers, [send us your request](mailto:gcstnmsd@microsoft.com).
+- For help with porting phone numbers inside the U.S or Canada, [send us your request](mailto:ptn@microsoft.com).
+
+### Outside of USA/Canada
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
### Outside the USA or Canada
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 2]
```diff
@@ -25,12 +25,16 @@ description: "When you are getting phone numbers or porting (transferring) numbe
...
+- For help with getting and porting phone numbers outside the U.S or Canada, [send us your request](mailto:ptneu@microsoft.com).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
- For help with getting and porting phone numbers outside the USA or Canada, [send us your request](mailto:ptneu@microsoft.com).
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
I accepted a couple minor copy edits to test a bug fix implemented by GitHub. @tonysmit still needs to review and merge.

**Comment by ErsancMS** [comment]
@tonysmit could you help us review and push this change?

**Pull request merged**